### ISSUE TITLE
Bugs: Improperly initialized values

### ISSUE THREAD

**Issue by BenjamenMeyer** [vegastrike/Vega-Strike-Engine-Source#346]
While reviewing https://github.com/vegastrike/Vega-Strike-Engine-Source/pull/345 I found several lines where a variable was not properly initialized. The basic pattern is:
```c++
type varName =
#ifdef SOME_DEFINED_VALUE
   type someOtherVar = theVarsValue;
   ...
#endif // SOME_DEFINED_VALUE
```
This is problematic from two perspectives as the variable (varName) is not actually initialized with its correct value in a verifiable manner:
1. If SOME_DEFINED_VALUE is defined, then it would receive the value of `someOtherVar` converted to its type.
2. If SOME_DEFINED_VALUE is *not* defined, then it'll either generate a compilation error and fail to compile (best case) or generate a warning/error and compile with a latent bug.

We should look at the git history of each file to try to determine if something was improperly removed in order to try to determine the correct values.

List of occurrences found in the PR:
- [ ] https://github.com/vegastrike/Vega-Strike-Engine-Source/pull/345#discussion_r528777404
- [ ] https://github.com/vegastrike/Vega-Strike-Engine-Source/pull/345#discussion_r529201188
- [ ] https://github.com/vegastrike/Vega-Strike-Engine-Source/pull/345#discussion_r529205347

**Comment by stephengtuggy** [comment]
These are probably causing some of the compile errors on Windows. (Since AL_whatever_it_is is undefined there for some reason.)

**Comment by BenjamenMeyer** [comment]
@stephengtuggy there's probably more, we just discovered these in the PR. We'll get there. Just will take some time and effort to find them all.

**Comment by stephengtuggy** [comment]
@BenjamenMeyer Yep. We might be able to run some sort of multi-line regex search to find the rest, I'm thinking. (I.e., search for an equals sign that has nothing but whitespace after it until the end of the line, then a `#if` at the beginning of the next line.)

**Comment by stephengtuggy** [comment]
I think I just fixed one of these in #545
